clean.cmd

```diff
@@ -0,0 +1,22 @@
+@ECHO OFF
```

Comment:
what's the purpose of this script? Isn't `git clean` sufficient?

Reply:
Oops, that was not intended to be included, I'll remove it

Reply:
But git clean does either too much or too less... this one removes exactly the bin and obj folders.

Reply:
e.g. It would also remove my Resharper configuration which I'm not allowed to commit.

Reply:
that's why I use `git clean -X`, which removes only files that match .gitignore entries, but does NOT touch uncommitted files. I have lots of uncommitted extra files in my ILSpy folder (test cases, diff files, TODO lists, notes, etc.), which some might argue is a bad thing, but I don't care :)

see https://git-scm.com/docs/git-clean

![image](https://github.com/user-attachments/assets/5d596ba4-6528-4b8b-9fee-d877e92540b4)

Usually, I combine that with `d` to remove empty directories and `q` for "quiet mode" and `f` is mandatory anyway. So, I use `git clean -Xdfq`.

Just configure your .gitignore correctly and `git clean` behaves exactly as it should.

Reply:
Just remember to always use upper-case X and you will be fine.

Reply:
If you think our .gitignore file is missing something, feel free to extend it to match your needs.